tests/xml/sirtfi_v2/good_sirtfi.yaml

```diff
@@ -0,0 +1,6 @@
+expected:
```

Comment:
I am sort of amused that having an `expected` key with nothing in it works. I wonder what that means? Empty array or empty object?

I guess there must be an appropriate handler for that case somewhere. You can just omit it entirely, as an alternative.

Reply:
Yes, I'll add `skip: true` as you suggested for the other sidecar file—that is much cleaner.

Reply:
Added this.